<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsWindow</class>
 <widget class="QWidget" name="SettingsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>864</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
background-color: rgb(222, 221, 218);</string>
  </property>
  <widget class="QWidget" name="widget_bg_color" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>621</width>
     <height>631</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="label_bg_color">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>158</y>
      <width>201</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid grey;
background-color: rgb(222, 221, 218);
color: rgb(94, 92, 100);</string>
    </property>
    <property name="text">
     <string>Background Color Settings</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_bg_color">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>621</width>
      <height>451</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid grey;
background-color: rgb(246, 245, 244);
color: rgb(154, 153, 150);
border-radius: 10px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_model_options" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>631</width>
     <height>311</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="groupBox_model_options">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>621</width>
      <height>261</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox {
border: 1px solid grey;
color: rgb(154, 153, 150);
background-color: rgb(237, 236, 232);
border-radius: 10px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QGroupBox" name="groupBox_projection">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>601</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
border: 1px solid grey;
color: rgb(154, 153, 150);
border-radius: 10px;
background-color: rgb(246, 245, 244);
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_parallel">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Parallel</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_projection_type</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_central">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>81</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Central</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_projection_type</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_projection_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>121</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(154, 153, 150);</string>
      </property>
      <property name="text">
       <string>Projection Type</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_vertex_options">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid grey;
color: rgb(94, 92, 100);
background-color: rgb(237, 236, 232);

</string>
     </property>
     <property name="text">
      <string>Vertices</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_edge_set">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
border: 1px solid grey;
background-color: rgb(246, 245, 244);
color: rgb(154, 153, 150);
border-radius: 10px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_edge_solid">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Solid</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_line_type</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_edge_dashed">
      <property name="geometry">
       <rect>
        <x>196</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Dashed</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_line_type</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_line_type">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(154, 153, 150);</string>
      </property>
      <property name="text">
       <string>Line Type</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_edge_width">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(154, 153, 150);</string>
      </property>
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_edge_color">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(154, 153, 150);</string>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_edge_width">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px solid grey;
background-color: rgb(246, 245, 244);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_edge_width">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">qproperty-alignment: 'AlignVCenter | AlignCenter';
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_edge_color">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 white, stop: 1 rgb(222, 221, 218));
	color: rgb(94, 92, 100);
	border: 1px solid grey;
	border-radius: 5px;
}

 QPushButton:hover {
	
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>Set Edge Color</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_vertex_options">
     <property name="geometry">
      <rect>
       <x>319</x>
       <y>40</y>
       <width>291</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
border: 1px solid grey;
background-color: rgb(246, 245, 244);
color: rgb(154, 153, 150);
border-radius: 10px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QRadioButton" name="radioButton_vertex_square">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Square</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_vertex_type</string>
      </attribute>
     </widget>
     <widget class="QRadioButton" name="radioButton_vertex_circle">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>20</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string>Circle</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup_vertex_type</string>
      </attribute>
     </widget>
     <widget class="QLabel" name="label_vertex_shape">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>51</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(154, 153, 150);</string>
      </property>
      <property name="text">
       <string>Shape</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_vertex_size">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(154, 153, 150);</string>
      </property>
      <property name="text">
       <string>Size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_vertex_color">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(246, 245, 244);
border:0px solid grey;
color: rgb(154, 153, 150);</string>
      </property>
      <property name="text">
       <string>Color</string>
      </property>
     </widget>
     <widget class="QDoubleSpinBox" name="doubleSpinBox_vertex_size">
      <property name="geometry">
       <rect>
        <x>204</x>
        <y>40</y>
        <width>51</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>ArrowCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">qproperty-alignment: 'AlignVCenter | AlignCenter';
color: rgb(0, 0, 0);
</string>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::NoButtons</enum>
      </property>
      <property name="minimum">
       <double>1.000000000000000</double>
      </property>
      <property name="maximum">
       <double>10.000000000000000</double>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider_vertex_size">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>101</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 0px solid grey;
background-color: rgb(246, 245, 244);</string>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>10</number>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
      <property name="tickInterval">
       <number>4</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_vertex_color">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>155</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 white, stop: 1 rgb(222, 221, 218));
	color: rgb(94, 92, 100);
	border: 1px solid grey;
	border-radius: 5px;
}

 QPushButton:hover {
	
	background-color: rgb(255, 255, 255);
}

QPushButton:pressed {
	background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1,
											  stop: 0 #dadbde, stop: 1 #f6f7fa);
}</string>
      </property>
      <property name="text">
       <string>Set Vertex Color</string>
      </property>
     </widget>
    </widget>
    <widget class="QCheckBox" name="checkBox_display_vertex_clicked_or_not">
     <property name="geometry">
      <rect>
       <x>453</x>
       <y>18</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid grey;
color: rgb(36, 31, 49);
background-color: rgb(237, 236, 232);


</string>
     </property>
     <property name="text">
      <string>Do not display</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_edge_options">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid grey;
color: rgb(94, 92, 100);
background-color: rgb(237, 236, 232);

</string>
     </property>
     <property name="text">
      <string>Edges</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_projection_type_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>180</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border: 0px solid grey;
color: rgb(94, 92, 100);
background-color: rgb(237, 236, 232);


</string>
     </property>
     <property name="text">
      <string>Projection Settings</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_model_options">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>221</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 0px solid grey;
background-color: rgb(222, 221, 218);
color: rgb(94, 92, 100);</string>
    </property>
    <property name="text">
     <string>3D Model Display Settings</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_apply_settings">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>830</y>
     <width>131</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(16, 112, 8);
	color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Apply Settings</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_save_settings">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>830</y>
     <width>140</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	
background-color: rgb(226, 140, 11);
	color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Set As Default</string>
   </property>
  </widget>
  <zorder>pushButton_apply_settings</zorder>
  <zorder>widget_bg_color</zorder>
  <zorder>widget_model_options</zorder>
  <zorder>pushButton_save_settings</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_vertex_type"/>
  <buttongroup name="buttonGroup_projection_type"/>
  <buttongroup name="buttonGroup_line_type"/>
 </buttongroups>
</ui>
